app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -280,6 +287,8 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         refreshAction?.let {
             browserToolbarView.view.addNavigationAction(it)
         }
+
+        browserToolbarView.view.invalidateActions()
```

Comment:
Wouldn't this call still throw a nullptr for the same reason?

Reply:
this function (`addTabletActions`) is only called when the `onUpdateToolbarForConfigurationChange` has been called which is null checked in the BaseBrowserFragment (since it is a private backing property).